### ISSUE TITLE
`make up` command errors out

### ISSUE THREAD

**Issue by Neha-Sinha2305** [fredrikhgrelland/vagrant-hashistack-template#60]
I get the following error when i run make up command:
```
Nehas-MacBook-Pro:vagrant-hashistack-template nehsin$ make up
==> default: Box 'fredrikhgrelland/hashistack' not installed, can't check for updates.
SSL_CERT_FILE= CURL_CA_BUNDLE= CUSTOM_CA= vagrant up --provision
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'fredrikhgrelland/hashistack' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0.5, < 0.6
==> default: Loading metadata for box 'fredrikhgrelland/hashistack'
    default: URL: https://vagrantcloud.com/fredrikhgrelland/hashistack
==> default: Adding box 'fredrikhgrelland/hashistack' (v0.5.0) for provider: virtualbox
    default: Downloading: https://vagrantcloud.com/fredrikhgrelland/boxes/hashistack/versions/0.5.0/providers/virtualbox.box
==> default: Box download is resuming from prior download progress
Download redirected to host: vagrantcloud-files-production.s3.amazonaws.com
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

OpenSSL SSL_read: Connection reset by peer, errno 54
make: *** [up] Error 1
```

**Comment by claesgill** [comment]
I recreate this scenario with no errors. However, I'm not on a Mac, but this might be fixed in the newer version of the box.

Can we consider this closed or is it still an issue? Maybe MacPD(@pdmthorsrud) can do a quick check?

**Comment by zhenik** [comment]
I tested it on Mac, do not have any errors.

```
 ~/src/github.com/fredrikhgrelland/vagrant-hashistack-template> make up                                                                                                                  ✔  08:06:09 
==> vagrant: A new version of Vagrant is available: 2.2.13 (installed version: 2.2.10)!
==> vagrant: To upgrade visit: https://www.vagrantup.com/downloads.html

==> default: Checking for updates to 'fredrikhgrelland/hashistack'
    default: Latest installed version: 0.7.1
    default: Version constraints: >= 0.7, < 0.8
    default: Provider: virtualbox
==> default: Box 'fredrikhgrelland/hashistack' (v0.7.1) is running the latest version.
SSL_CERT_FILE= CURL_CA_BUNDLE= CUSTOM_CA= vagrant up --provision
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Cloning VM...
==> default: Matching MAC address for NAT networking...
==> default: Checking if box 'fredrikhgrelland/hashistack' version '0.7.1' is up to date...
==> default: Setting the name of the VM: vagrant-hashistack-template_default_1605078390622_54216
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 8500 (guest) => 8500 (host) (adapter 1)
    default: 4646 (guest) => 4646 (host) (adapter 1)
    default: 8200 (guest) => 8200 (host) (adapter 1)
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default: 
    default: Guest Additions Version: 6.0.10
    default: VirtualBox Version: 6.1
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /vagrant => /Users/zhenik/src/github.com/fredrikhgrelland/vagrant-hashistack-template
==> default: Running provisioner: shell...
    default: Running: inline script
    default: You may add /vagrant/dev/vagrant/bootstrap/pre_ansible.sh if you want something to happen here
==> default: Running provisioner: ansible_local...
    default: Running ansible-playbook...

PLAY [all] *********************************************************************

TASK [Gathering Facts] *********************************************************
ok: [default]

TASK [Check if .env_override is provided when tests are in CI] *****************
ok: [default] => {}

MSG:

Checking if env override process was successful for ci test
```

**Comment by claesgill** [comment]
Let's close it then 🎉
